src/uno/RoslynQuoter.Shared/UnoPlatformShell.xaml

```diff
@@ -0,0 +1,327 @@
+										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+										  Margin="{TemplateBinding Padding}"
+										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}" />
+					</Grid>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<!-- Primary Button Style -->
+	<Style x:Key="PrimaryButtonStyle"
+		   TargetType="Button">
+
+		<Setter Property="Background"
+				Value="Transparent" />
+		<Setter Property="BorderThickness"
+				Value="0" />
+		<Setter Property="Padding"
+				Value="16" />
+		<Setter Property="HorizontalAlignment"
+				Value="Stretch" />
+		<Setter Property="VerticalAlignment"
+				Value="Stretch" />
+		<Setter Property="VerticalContentAlignment"
+				Value="Stretch" />
+		<Setter Property="HorizontalContentAlignment"
+				Value="Stretch" />
+
+		<!-- TextBlock Styling -->
+		<Setter Property="Foreground"
+				Value="{StaticResource Color02Brush}" />
+		<Setter Property="FontSize"
+				Value="14" />
+		<Setter Property="FontWeight"
+				Value="SemiBold" />
+
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="Button">
+
+					<Grid x:Name="RootGrid"
+						  Background="{TemplateBinding Background}">
+
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualState x:Name="Normal" />
+								<VisualState x:Name="PointerOver" />
+								<VisualState x:Name="Pressed">
+									<VisualState.Setters>
+										<Setter Target="ContentPresenter.Opacity"
+												Value="0.6" />
+									</VisualState.Setters>
+								</VisualState>
+								<VisualState x:Name="Disabled">
+									<VisualState.Setters>
+										<Setter Target="ContentPresenter.Opacity"
+												Value="0.6" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+						<!-- Label -->
+						<TextBlock x:Name="ContentPresenter"
+								   Text="{TemplateBinding Content}"
+								   VerticalAlignment="Center"
+								   HorizontalAlignment="Center"
+								   Margin="{TemplateBinding Padding}" />
+					</Grid>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+	
+	<!-- UnoPlatformShell -->
+	<Style TargetType="UnoPlatformShell">
+		
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="UnoPlatformShell">
+
+					<Grid>
+						<Grid.RowDefinitions>
+							<RowDefinition Height="auto" />
+							<RowDefinition Height="*" />
+						</Grid.RowDefinitions>
+
+						<!-- Header -->
+						<Grid Background="{StaticResource Color01Brush}"
+							  Height="90"
+							  Grid.Row="0">
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="Auto" />
+								<ColumnDefinition Width="*" />
+								<ColumnDefinition Width="Auto" />
+							</Grid.ColumnDefinitions>
+
+							<!-- Uno Logo -->
+							<Image Source="ms-appx:///Assets/UnoLogo.scale-200.png"

```

Comment:
you shouldn't have to specify the .scale-200   just UnoLogo.png will work.

Reply:
Done. Change the name of the Asset itself to UnoLogo.png